docs/README.md

```diff
@@ -274,6 +274,39 @@ Endpoints define the entry points of Couper. The mandatory *label* defines the p
 | `path`|<ul><li>changeable part of upstream URL</li><li>changes the path suffix of the outgoing request</li></ul>|
 |[**`access_control`**](#access_control_attribute)|sets predefined `access_control` for `endpoint`|
 |[**`backend`**](#backend_block) block |configures connection to a local/remote backend service for `endpoint`|
+|[**`remove_query_params`**](#query_params)|<ul><li>a list of query parameters to be removed from the upstream request URL</li></ul> |
+|[**`set_query_params`**](#query_params)|<ul><li>key/value(s) pairs to set query parameters in the upstream request URL</li></ul> |
+|[**`add_query_params`**](#query_params)|<ul><li>key/value(s) pairs to add query parameters to the upstream request URL</li></ul> |
+
+#### Query parameter <a name="query_params"></a>
+
+Couper offers three methods to manipulate the query parameter. The methods are
+executed in the following order:
```

Comment:
We should mention that this is unrelated to the order within the configuration file.